docker-compose.yaml

```
@@ -40,22 +40,22 @@ services:
     env_file:
       - ./.env
     environment:
-      ELASTICSEARCH_ENDPOINT: http://sitesearch:9200/
+      OPENSEARCH_ENDPOINT: http://sitesearch:9200/
       BASE_URL: https://www.giantswarm.io/blog
 
 
   sitesearch:
     restart: no
     platform: linux/amd64
-    image: gsoci.azurecr.io/giantswarm/elasticsearch:6.8.13
+    image: opensearchproject/opensearch:2.17.1
     ports:
       - "9200:9200"
     environment:
       cluster.name: sitesearch
       node.name: sitesearch1
       discovery.type: single-node
       bootstrap.memory_lock: "false"
-      ES_JAVA_OPTS: -Xms256m -Xmx256m
-      ES_HOME: /usr/share/elasticsearch
+      OPENSEARCH_JAVA_OPTS: -Xms256m -Xmx256m
```

Let's try to use the same options as for the production sitesearch app:
```suggestion
      OPENSEARCH_JAVA_OPTS: "-XX:+UseContainerSupport -XX:InitialRAMPercentage=40.0 -XX:MinRAMPercentage=30.0 -XX:MaxRAMPercentage=80.0"
```